src/DTOs/QuestionDTO.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+
+namespace QuizApp\DTOs;
+
+
+class QuestionDTO
+{
+    private $question;
```

Comment:
a question has a question? :)) $question->getQuestion() is counter-intuitive; call it text maybe

Reply:
Yes, Master